R/finalize_plot.R

```diff
@@ -201,18 +219,23 @@ finalize_plot <- function(input_plot = NULL,
     clip = "on"
   )
 
+  # extract height of legend object within ggplot plot
+  legend_height <- grid::convertUnit(ggpubr::get_legend(processed_plot)$heights[[3]],
```

Comment:
Can we move this up into the initialization section?

Reply:
Yes, sure. Done.